frontend/util.lua

```diff
@@ -597,4 +597,9 @@ function util.shell_escape(args)
     return table.concat(escaped_args, " ")
 end
 
+table.clear = function(t)  -- luacheck: ignore 122
```

Comment:
I was going to complain about setting a fake language primitive myself actually; I hadn't realized luacheck had flagged it already.

Reply:
Oh wait, I *did* complain about it. :-P

Reply:
Oh, I thought table.clear() was preferred. If not, I will move it to util.